### DISCUSSION TITLE
Formatter - Clang how to add in CP Editor

### DISCUSSION THREAD

**Discussion by yousuf7902** [cpeditor/cpeditor#1104] [Q&A] [answered]
Normally i installed llvm and after that how I add clang format in cp editor. I can't understand help me.
![cp editor](https://user-images.githubusercontent.com/70694745/191002172-5c86a0f8-aee9-4a52-ac9e-92ef0a4ec29d.png)

**Comment by ouuan** [comment, marked as answer]
You need to either add the path to the `clang-format` executable file in the `PATH` environment variable, or set the path to the `clang-format` executable file in the "Clang Format Program" setting of CP Editor.
